parsing/ast_mapper.ml

```diff
@@ -843,7 +843,6 @@ module PpxContext = struct
         lid "principal", make_bool !Clflags.principal;
         lid "transparent_modules", make_bool !Clflags.transparent_modules;
         lid "unboxed_types", make_bool !Clflags.unboxed_types;
-        lid "unsafe_string", make_bool !Clflags.unsafe_string;
```

Comment:
I think it would be better if this was still included and constantly returned `false`.

Reply:
Done